src/main/java/com/project/momo/security/filter/JwtFilter.java

```diff
@@ -0,0 +1,46 @@
+package com.project.momo.security.filter;
+
+import com.project.momo.security.jwt.TokenProvider;
+import com.project.momo.utils.ResponseManager;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.util.StringUtils;
+import org.springframework.web.filter.GenericFilterBean;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@RequiredArgsConstructor
+public class JwtFilter extends GenericFilterBean {
+
+    private final String AUTHORIZATION_HEADER = "Authorization";
```

Comment:
```suggestion
    private static final String AUTHORIZATION_HEADER = "Authorization";
```
상수는 static으로 정의합니다.

또한 Springframework에서는 HttpHeaders.AUTHORIZATION와 같은 상수값을 제공합니다.
굳이 새로운 상수를 정의하기보다는 이미 제공하고 있는 상수가 있는 경우 사용하는편이 더 좋다고 생각드네요 🤔